packages/contentstack-bulk-publish/src/commands/cm/entries/publish.js

```diff
@@ -22,7 +22,7 @@ class PublishEntriesCommand extends Command {
       entriesFlags['publish-all-content-types'] || entriesFlags.publishAllContentTypes || false;
     entriesFlags.apiVersion = entriesFlags['api-version'] || '3';
     entriesFlags.includeVariants = entriesFlags['include-variants'] || entriesFlags.includeVariants || false;
-    entriesFlags.entryUid = entriesFlags['entry-uid'] || entriesFlags.entryUid;
+    entriesFlags.entryUid = entriesFlags['entry-uid'] || entriesFlags.entryUid || false;
```

Comment:
@sunil-lakshman , don't think false required here as entry-uid flag is of string type. pls handle it properly like this
if(!entriesFlags.entryUid){
//logic
}